sklearn/compose/_column_transformer.py

```diff
@@ -136,6 +136,11 @@ class ColumnTransformer(TransformerMixin, _BaseComposition):
         sparse matrix or a dense numpy array, which depends on the output
         of the individual transformers and the `sparse_threshold` keyword.
 
+    transformers_output_ : dict
+        A dictionary from transformer names to slices to track the results
+        of each transformer. The slices are the positions in the stacked
+        results of the output of each particular step in `transformers`.
```

Comment:
```suggestion
        A dictionary from transformer names to slices, where
        each slice corresponds to indices in the
        transformed output. This is useful to inspect which
        transformer is responsible for which transformed feature.
```

Reply:
I think `slice` needs to be hyperlinked to Python docs since they are not common.